blocks/OSC/src/OscMessage.h

```diff
@@ -58,11 +58,13 @@ namespace cinder { namespace osc {
 		int32_t getArgAsInt32( int index, bool typeConvert = false ) const;
 		float getArgAsFloat( int index, bool typeConvert = false ) const;
 		std::string getArgAsString( int index, bool typeConvert = false ) const;
-		
+		ci::Buffer getArgAsBlob(int index, bool typeConvert = false) const;
```

Comment:
Need the one space padding within the parens `ci::Buffer getArgAsBlob( int index, bool typeConvert = false )`